SlowCheetah.JDT/JsonTransformation.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) Sayed Ibrahim Hashimi. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See  License.md file in the project root for full license information.
+
+namespace SlowCheetah.JDT
+{
+    using System;
+    using Newtonsoft.Json.Linq;
+
+    /// <summary>
+    /// Represents a transformation based on a JSON file using JDT
+    /// </summary>
+    public class JsonTransformation
```

Comment:
JsonTransformation now operates with streams of files directly, always returning a stream for the user to save